cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,130 @@
+- Regularly scanning our environment using vendors designated by PCI as [Approved Scanning Vendors (ASVs)](https://www.pcidssguide.com/what-is-a-pci-approved-scanning-vendor-asv/) to ensure our continued compliance with PCI DSS 3.2.1, and correcting issues as quickly as possible.
+- Regularly scanning our environment and software for known security vulnerabilities and applying updates and security patches in a timely manner.
+- Implementing [data loss prevention (DLP)](https://pcidss.com/listing-category/data-loss-protection-dlp).
+- Logging cluster actions and events, redacting sensitive information in audit logs, and retaining audit logs according to the [logging requirements of PCI DSS](https://listings.pcisecuritystandards.org/documents/Effective-Daily-Log-Monitoring-Guidance.pdf).
+
+A comprehensive list of all actions that Cockroach Labs takes to ensure compliance with PCI DSS 3.2.1 is beyond the scope of this document. For more information, contact your Cockroach Labs account team.
+
+Cockroach Labs automatically and regularly applies security and stability updates to {{ site.data.products.dedicated }} clusters. These updates include, but are not limited to, the cluster's CockroachDB runtimes and APIs, the operating systems of cluster nodes, and management utilities. You are notified about upcoming cluster maintenance before it happens, when it starts, and when it completes. Refer to [Monitoring and Alerting](/docs/stable/monitoring-and-alerting.html). You can also read the release notes for [CockroachDB](/docs/releases/index.html) and for [{{ site.data.products.db }}](/docs/releases/cloud.html).
+
+Compliance is a shared responsibility. Be sure to read [Responsibilities of the customer](#responsibilities-of-the-customer) to ensure that your cluster is compliant.
+
+## Responsibilities of the customer
+
+To ensure that a {{ site.data.products.dedicated }} cluster complies with PCI DSS 3.2.1, you must enable and configure the following cluster settings and features:
+
+- The cluster must be created as a {{ site.data.products.dedicated }} cluster. A private cluster's nodes have no public IP addresses, and its egress traffic moves over private subnets and through a highly-available NAT gateway that is unique to the cluster. An existing cluster cannot be migrated to be a private cluster.
+- [Customer-Managed Encryption Keys (CMEK)](cmek.html) must be enabled on the cluster. CMEK protects data at rest in a CockroachDB Dedicated cluster using a cryptographic key that is entirely within your control, hosted in a supported key-management system (KMS) platform.
+- [Egress Perimeter Controls](egress-perimeter-controls.html) must be enabled, Egress Perimeter Controls ensure that cluster egress operations, such as cluster backups, are restricted to a list of specified external destinations.
+- [Cluster log export](export-logs.html)'s redaction feature must be enabled to prevent the exposure of sensitive data in exported logs.
```

Comment:
```suggestion
- [Cluster log export](export-logs.html)'s redaction feature must be enabled to prevent the exposure of sensitive data in logs exported to customer's instance of AWS Cloudwatch or GCP Cloud Logging.
```